tools/benchmark/src/main/java/com/datadog/benchmark/exporter/DatadogMetricExporter.kt

```diff
@@ -46,6 +46,8 @@ internal class DatadogMetricExporter(datadogExporterConfiguration: DatadogExport
     }
 
     companion object {
+        private const val DEFAULT_MODEL_NAME = "unknown model"
```

Comment:
maybe we should call these constants `UNKNOWN_...`

Reply:
Actually the text here should be displayed on the web, as the tag value such as "device: unknown model", 
that's why I use plain text, do you prefer we use upper case with underscore on this?

<img width="744" alt="image" src="https://github.com/user-attachments/assets/053dd9f9-a5f1-4c33-b56e-45df63c960b2">

Reply:
nope I was referring to the variable name UNKNOWN instead of DEFAULT as prefix

Reply:
OK,  then you are right, I updated the variable name, and now all attributes share the same unknown value.